ansible (#50)
VeritasOS/netbackup-api-code-samples: snippets/ansible/README.md

> ### NetBackup API Code Samples for Ansible

Reviewer: Let's start headings at level 1 instead of level 3, please.

```suggestion
# NetBackup API Code Samples for Ansible
```

This directory contains code samples to invoke NetBackup REST APIs using ansible.

#### Disclaimer

These scripts are only meant to be used as a reference. If you intend to use them in production, use it at your own risk.

#### Pre-requisites:

- NetBackup 8.1.1 or higher

> - ansible 2.x

Reviewer: Capitalize proper nouns.

```suggestion
- Ansible 2.x
```

#### Executing the snippets in ansible

These are tasks meant to be part of a larger playbook.   I use them with the following syntax

> tasks:
> -name: name of tasks
>   include_task: tasklocation/taskname

Reviewer: Mark for display as code, and make it valid YAML syntax.

```suggestion
```yaml
tasks:
- name: name of tasks
  include_task: tasklocation/taskname
\```
```

> Vars for ansible are defined in a inventory file for use in plays
> vars created during plays are used in future ones like login.yml.

Reviewer: Markdown renders this as one sentence. And passive voice makes it unclear whose responsibility it is to perform the actions described here.

```suggestion
Define variables for Ansible in an inventory file for use in plays.
Variables registered during plays like login.yaml are used in subsequent plays.
```
